packages/riipen-ui/src/components/Chip.jsx

```diff
@@ -2,13 +2,15 @@ import clsx from "clsx";
 import PropTypes from "prop-types";
 import React from "react";
 
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faTimes } from "@fortawesome/free-solid-svg-icons";
-
 import ThemeContext from "../styles/ThemeContext";
 
 class Chip extends React.Component {
   static propTypes = {
+    /**
+     * The elements to render inside the pill
```

Comment:
Small nitpick, but add a period to the end of this (and all subsequent prop comments) since they are rendered in the docs as full sentences.